<?xml version="1.0"?>

<launch>

  <node ns="object_categorization" name="object_categorization" pkg="cob_object_categorization" type="object_categorization" output="screen">
    <remap from="input_pointcloud_segments" to="/object_segmentation/segmented_object"/>  <!-- simple segmentation provided by this package -->
    <!--remap from="input_pointcloud_segments" to="/tabletop_object_cluster/cluster_array"/-->   <!-- segmentation of cob_table_object_cluster -->
    <!--remap from="input_color_image" to="/cam3d/rgb/image_color"/-->
    <!--remap from="input_pointcloud_camera_info" to="/cam3d/depth_registered/camera_info"/-->
    <remap from="input_color_image" to="/camera/rgb/image_color"/>
    <remap from="input_pointcloud_camera_info" to="/camera/depth_registered/camera_info"/>
    
    <param name="mode_of_operation" type="int" value="2"/>		<!-- 2=Hermes detect,  3=Hermes training -->
    <param name="object_name" type="string" value="shoe_black"/>	<!-- for Hermes: the object model that shall be loaded or trained on startup -->
    <param name="start_categorization_on_startup" type="bool" value="false"/>	<!-- if true, categorization publishes continuously from startup, if false, results may only obtained via the action interface -->
  </node>

</launch>
